Musician Module/Controls/Instrument/Instrument.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using Blish_HUD.Modules.Musician.Domain.Values;
+using Microsoft.Xna.Framework;
+namespace Blish_HUD.Modules.Musician.Controls.Instrument
+{
+    public enum InstrumentSkillType
+    {
+        None,
+        LowNote,
+        MiddleNote,
+        HighNote,
+        IncreaseOctaveToMiddle,
+        IncreaseOctaveToHigh,
+        DecreaseOctaveToLow,
+        DecreaseOctaveToMiddle,
+        StopPlaying
+    }
+    public abstract class Instrument
+    {
+        public bool IsInstrument(string instrument) {
+            return this.GetType().Name.ToLower().Trim() == instrument.ToLower().Trim();
```

Comment:
Probably easier to just do `string.Equals(this.GetType().Name, instrument, StringComparison.OrdinalIgnoreCase);` instead of ensuring you've ToLower()'d everything.  Also, did you encounter a problem with the type name having white space at the end?  Seems odd to `.Trim()` it.

Reply:
yes I always use that. My bad